onyo/commands/new.py

```diff
@@ -14,16 +14,31 @@
 logging.basicConfig()
 logger = logging.getLogger('onyo')
 
+reserved_characters = [".", "_"]
+
 
 def build_commit_cmd(file, git_directory):
-    return ["git -C " + git_directory + " commit -m", "\'new " + file + "\'"]
+    return ["git -C " + git_directory + " commit -m", "\'new \"" + file + "\"\'"]
+
+
+def read_new_field(field_description):
+    # read word for field from keyboard
+    field = str(input(field_description))
+    # if field contains one of the reserved characters, read a new word.
```

Comment:
If the field contains a reserved character, it should bark at the user, inform them which characters are reserved, and tell them to try again.

Nitpick: `word` should be normalized